modules/ROOT/pages/http-caching-policy.adoc

```diff
@@ -6,241 +6,419 @@ endif::[]
 
-This is the maximum number of responses that can be present in the cache at the same time. The criteria for removing the responses from the cache when this maximum is surpassed is FIFO. This configuration is ignored in Runtime Manager.
+=== Persistent Cache
 
-This field is also mandatory and has a default value of 10000 entries.
+A persistent cache enables the stored entries in the cache to persist if the Mule runtime engine (Mule) is restarted.
 
-*Note:* When using the Runtime Manager ObjectStore v2, this is not enforced.
+When you upgrade a Mule version that has the HTTP Caching policy configured to use the persistent store, the policy tries to maintain the entries stored by the previous Mule version. However in a worst-case scenario, the entries in the cache are invalidated and the cache is re-populated when new requests arrive. This manipulation of the entries in the cache occurs automatically and is invisible to the user.
+//When you upgrade to a Mule version?--Lorena
 
-=== Entry Time To Live
 
-The Time to live (TTL) value is measured in seconds.
+== HTTP Caching Directives
 
-This parameter represents how many seconds a response is stored in the cache, and once this time elapses, that the request is removed from the cache. When a new request arrives, the response is not present in the cache, and a new response must be computed.
-When using the policy with Object Store v2 in Runtime Manager, the edition of this value won't be correctly reflected, in this case it's recommended to remove the policy and object store and apply a new one.
+To obtain more control over the cache, the HTTP Caching policy interprets some of the HTTP directives from the https://tools.ietf.org/html/rfc7234[RFC-7234] protocol by taking the following headers into account:
 
-*Note:* This is a mandatory parameter and its default value is ten minutes (600 seconds).
+* Cache-Control
+* Expires
+* Date
+* Age
 
-=== Distributed
+//if they are syntax, format them in monospace and exactly mimic the onscreen capitalization.
 
-When running a cluster or several workers in Runtime Manager, the values stored in the cache can be shared between the different nodes. To accomplish this, this option must be checked, otherwise each node has its own cache. In the case of Runtime Manager, the object store v2 must be enabled for this option to work.
+=== Cache-Control
 
-By default this option is unchecked and there maybe a small penalty when distributing the cache, but negligible.
+The Cache-Control header can exist either in the request or in the response. Possible values for the header, which can be combined separated by commas, include:
 
-=== Persist Cache
 
-If this box is checked the values stored in the cache persist after a runtime restart. If this option is used in Runtime Manager, enable the object store V2 on the application’s configuration in Runtime Manager.
+* In requests:
 
-By default this option is unchecked.
+** no-cache
+//These are values right?Lorena
```

Comment:
@priyaayyar Yes, exactly. `no-cache` and `no-stores` are possible values for the request, while the others below are possible values for the response.

Reply:
Ok done!